tutorials/evals/evaluate_function_calling.ipynb

```diff
@@ -0,0 +1,2522 @@
+{
```

Comment:
<p>Imports breaking here.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Arize-ai/phoenix/pull/3881/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
I'm pretty sure this is due to this PR adding the eval templates that the notebook in the same PR is trying to import. So the phoenix[evals] package used by the notebook won't include those new templates without this PR.

Thinking I should split this into 2 PRs, one to add the template and one to add the notebook. Sound good?

Reply:
Connected separately - I was wrong, this came from the template code not being present in `evals.__init__.py` and `default_templates.py`